plugins/search-backend-module-elasticsearch/src/alpha.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  coreServices,
+  createBackendModule,
+} from '@backstage/backend-plugin-api';
+import { loggerToWinstonLogger } from '@backstage/backend-common';
+import { searchEngineRegistryExtensionPoint } from '@backstage/plugin-search-backend-node/alpha';
+
+import {
+  ElasticSearchHighlightConfig,
+  ElasticSearchQueryTranslatorOptions,
+  ElasticSearchConcreteQuery,
+  ElasticSearchCustomIndexTemplate,
+  ElasticSearchCustomIndexTemplateBody,
+  ElasticSearchQueryTranslator,
+  ElasticSearchSearchEngine,
+} from './engines';
```

Comment:
there's some code missing here right?

Reply:
what code were you thinking of? 😬 we removed a couple of imports as part of https://github.com/backstage/backstage/pull/16710/commits/be68ffd6ad13db78b4f4be6780c2defe7d2bf0ba maybe that fixes what you refer to?